optimade/validator/validator.py

```diff
@@ -561,8 +608,10 @@ def get_available_endpoints(self, base_info):
     @test_case
     def get_endpoint(self, request_str, optional=False):
         """ Gets the response from the endpoint specified by `request_str`. """
-        request_str = f"/{request_str}".replace("\n", "")
+
+        request_str = request_str.replace("\n", "")
```

Comment:
So it will _never_ need to be prepended with an `/` anymore? Should we check this and make sure we end up with the right string structure?

Reply:
`urllib.parse.urljoin` is now being used which won't care about these slashes.

Reply:
Sweet!